Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -275,6 +294,61 @@ def _update_stats_model(self):
                 row.append(item)
             model.appendRow(row)
 
+    def _update_class_selection(self):
+        if is_discrete(self.train_data.domain.class_var):
+            self.cbox.setVisible(True)
+            values = self.train_data.domain.class_var.values
+            self.class_selection_combo.clear()
+            self.class_selection_combo.addItem("(None)")
+            self.class_selection_combo.addItems(values)
+
+            class_index = 0
+            if self.class_selection != '(None)' and self.class_selection != 0:
+                class_map = {val: i for i, val in enumerate(self.train_data.domain.class_var.values)}
+                class_index = class_map[self.class_selection]+1
+
+            self.class_selection_combo.setCurrentIndex(class_index)
+            self.previous_class_selection = "(None)"
+        else:
+            self.cbox.setVisible(False)
+
+    def one_vs_rest(self, res):
+        if self.class_selection != '(None)' and self.class_selection != 0:
+            class_map = {val: i for i, val in enumerate(self.train_data.domain.class_var.values)}
+            class_ = class_map[self.class_selection]
+            actual = res.actual == class_
+            predicted = res.predicted == class_
+            return Results(
+                nmethods=1, domain=self.train_data.domain,
+                actual=actual, predicted=predicted)
+        else:
+            return res
+
+    def _select_class(self):
+        if self.previous_class_selection == self.class_selection:
+            return
+        
+        results = list(split_by_model(self.results))
+
+        items = [(key, input) for key, input in self.learners.items()]
+        learners = [input.learner for _, input in items]
+
+        class_var = self.train_data.domain.class_var
+        if is_discrete(class_var):
+            test_stats = classification_stats
```

Comment:
Why do you need test_stats instead of calling classification_stats (regression_stats) directly?